ivy/container/layers.py

```diff
@@ -350,7 +350,7 @@ def dropout(
 
     @staticmethod
     def static_dropout1d(
-        x: ivy.Container,
+        x: Union[ivy.Array, ivy.NativeArray, ivy.Container],
```

Comment:
This instance method only accepts `ivy.Container` as input

Reply:
Sorry for the late reply. I agree that your review is correct, but I've seen many conflicting examples (i.e. elementwise) where a `ivy.Container.static_` function does accept both `ivy.Array` and `ivy.Container` as a valid type hint. In `ivy.ContainerBase.cont_multi_map_in_function`, we even assert that a container must exist in the arguments. So are the type hints wrong in these examples?

Reply:
Only the mentioned line should be changed as the function accepts one input as array and for the container. This means this static method can't accept the `ivy.Array` input and aswell `ivy.NativeArray` inputs.